src/Command/SF1514/SAMLTokenCommand.php

```diff
@@ -0,0 +1,197 @@
+<?php
+
+/**
+ * This file is part of itk-dev/serviceplatformen.
+ *
+ * (c) 2020 ITK Development
+ *
+ * This source file is subject to the MIT license.
+ */
+
+namespace ItkDev\Serviceplatformen\Command\SF1514;
+
+use GuzzleHttp\Client;
+use Http\Adapter\Guzzle6\Client as GuzzleAdapter;
+use Http\Factory\Guzzle\RequestFactory;
+use ItkDev\AzureKeyVault\Authorisation\VaultToken;
+use ItkDev\AzureKeyVault\KeyVault\VaultSecret;
+use ItkDev\Serviceplatformen\Certificate\AzureKeyVaultCertificateLocator;
+use ItkDev\Serviceplatformen\Certificate\CertificateLocatorInterface;
+use ItkDev\Serviceplatformen\Certificate\FilesystemCertificateLocator;
+use ItkDev\Serviceplatformen\Service\SF1514\SF1514;
+use ItkDev\Serviceplatformen\Service\SoapClient;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Exception\InvalidOptionException;
+use Symfony\Component\Console\Input\InputDefinition;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class SAMLTokenCommand extends Command
+{
+    protected static $defaultName = 'serviceplatformen:sf1514:saml-token';
+
+    private array $inputOptions = [];
+
+    protected function configure()
+    {
+        $inputOptions = [
+            new InputOption('production', null, InputOption::VALUE_NONE, 'production mode'),
+            new InputOption('certificate', null, InputOption::VALUE_REQUIRED, 'path to certificate'),
+            new InputOption('passphrase', null, InputOption::VALUE_OPTIONAL, 'passphrase to certificate'),
+            new InputOption('authority-cvr', null, InputOption::VALUE_REQUIRED, 'authority cvr', '55133018'),
+            new InputOption('sts-applies-to', null, InputOption::VALUE_REQUIRED, 'service SAML token should apply to'),
+        ];
+        $this->setDefinition(new InputDefinition($inputOptions));
+
+        $this->setDescription('Helper script for testing “SF1514: Sikkerhed - Hent Token fra Security Token Service”.');
```

Comment:
Do we really need a command for this?

Reply:
Not sure, used it as an intermediate step towards the sf1500 command. I suppose it could be used to simply test new certificates?

Reply:
Removed.